### ISSUE TITLE
no_prefix + encodes special chars in the URL

### ISSUE THREAD

**Issue by gjovanov** [nuxt-modules/i18n#983]
Hi there,
we have a scenario for this config:
```javascript
i18n: {
    langDir: 'lang/',
    fallbackLocale: 'en',
    lazy: true,
    strategy: 'no_prefix',
    locales: [
      { code: 'en', iso: 'en-US', name: 'English', file: 'en.js' },
      { code: 'de', iso: 'de-DE', name: 'Deutsch', file: 'de.js' }
    ]
  }
```

and route:
- `/@/_username`

it encodes the the route path as:
`/%40/:username`

How can we prevent the URL encoding so that the char `@` remains?

Many thanks.

**Comment by rchl** [comment]
It's Nuxt doing that, not this module.

See https://github.com/nuxt/nuxt.js/pull/8421 and related fixes.

I believe your issue is still not fixed though (according to me quickly checking `nuxt-edge`)

@pi0

**Comment by gjovanov** [comment]
Tnx for clarifying @rchl ... For now, as a workaround, we have replaced the '@' char with '-' (which doesn't get encoded) in the URL routes, until this get's fixed in NuxtJS.

Btw shall I close this bug or you want to keep it open for the sake of transparency?

**Comment by pi0** [comment]
Hi. Issue with `@` character should be fixed by [v2.14.10](https://github.com/nuxt/nuxt.js/releases/tag/v2.14.10)
